gulpfile.js

```diff
@@ -1,16 +1,39 @@
-var elixir = require('laravel-elixir');
-
-/*
- |--------------------------------------------------------------------------
- | Elixir Asset Management
- |--------------------------------------------------------------------------
- |
- | Elixir provides a clean, fluent API for defining some basic Gulp tasks
- | for your Laravel application. By default, we are compiling the Sass
- | file for our application, as well as publishing vendor resources.
- |
+var gulp = require('gulp');
+
+/**
+ * Copy any needed files.
+ *
+ * Do a 'gulp copyfiles' after bower updates
  */
+gulp.task("copyfiles", function() {
```

Comment:
Combine css library files to `public/backend/css/vendor.css', combine js file to `public/backend/js/vendor.js`